pkg/operator/helm.go

```diff
@@ -87,7 +87,8 @@ func renderChartsArchive(deployedVersionArchive string, downstreamName string, k
 				}
 			}
 
-			if info.Name() == "kustomization.yaml" {
+			_, err = os.Stat(sourceChartsDir)
```

Comment:
charts dir is `base/charts`. inside this dir is `base/charts/[chart name]` for each chart. are you sure you dont want to check each chart individually? if not why iterate at all and just check this after line 70?

https://github.com/replicatedhq/kots/pull/2856/files#diff-685325e8ac6019af11c7f04174b7ae09e382a86ae9d0e59563788a5a936354e5R70

Reply:
Ah, I'll have another look, thanks for catching this

Reply:
@emosbaugh can you take another look, I added the relative path to the source charts dir. I think this is what you meant initially.

Reply:
lgtm